bin/plotting/pycbc_page_dq_table

```diff
@@ -39,10 +39,13 @@ frac_livetimes = [lt / total_livetime for lt in livetimes]
 state_names = list(dq_states.values()) + ['Total']
 columns = [state_names, livetimes, frac_livetimes]
 columns = [np.array(c) for c in columns]
-col_names = ['DQ State', 'Livetime', 'Fractional Livetime']
+col_names = ['DQ State', 'Livetime', '% of Livetime']
+
+format_strings = [None, '#.#', '#.##%']
```

Comment:
If I understand correctly the '%' here automatically converts a fraction into a percentage (and prints '%' after it).